eth/tracers/internal/tracetest/calltrace_test.go

```diff
@@ -47,9 +47,10 @@ type callContext struct {
 
 // callLog is the result of LOG opCode
 type callLog struct {
-	Address common.Address `json:"address"`
-	Topics  []common.Hash  `json:"topics"`
-	Data    hexutil.Bytes  `json:"data"`
+	Address  common.Address `json:"address"`
+	Topics   []common.Hash  `json:"topics"`
+	Data     hexutil.Bytes  `json:"data"`
+	Position int            `json:"position"`
```

Comment:
Please add a comment to indicate position is with respect to sub calls.

Reply:
Sorry this is not the right place, I mean in the main tracer file not test file

Reply:
Added